Support/create-a-pdf-from-ctl-web-page.md

```diff
@@ -0,0 +1,16 @@
+{{{
+  "title": "Create a PDF from a Web Page on CTL.io",
+  "date": "1-22-2019",
+  "author": "Lyza Latham",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+### Overview
+If you should ever need to produce a printable PDF from any page on the www.ctl.io website, we have provided a handy tool.
+
+### Scroll Down to the Footer
+In the footer of every page, there's a text link under *Support* that reads *PDF This Page*. Click on that link and the CenturyLink wheel icon will start to spin, indicating that the PDF is being written. Once created, the PDF will download automatically to your desktop.
+
+### Resolution vs. File Size
+You may notice that the resolution on the PDF is not very high. That's due to the tradeoff between resolution and file size. The higher the resolution, the more massive the PDF. We decided to strike a balance and deliver a PDF that's a bit on the low end for resolution, but delivers a file size that's more manageable. Even so, many PDF files created will be far too large to email. Pages that are extremely dense in content, or have many graphics and charts, will deliver large PDFs. In such cases, we recommend you simply zip the file for transmittal.
```

Comment:
I think we need just need to get rid of `Resolution vs. File Size` section. It was valid when we were trying to balance them. But we do not do it any more.
In general we are using fixed minimum readable resolution (Do we need to mention it? May be it is obvious). Size of file depends now only on the size of content we converting, no altering of it. That is given.

Some pages are of significant size, which can produce bigger than mailable size PDF. If needed to email, use zip to reduce the size of file.

The last statement we may put just as a warning since we are not responsible for that any more.

But this is just my thought about